contents/testing/security-testing.md

```diff
@@ -0,0 +1,91 @@
+<frontmatter>
+  title: Introduction to Security Testing
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Security Testing
+
+**Author(s): [Ahmed Bahajjaj](https://github.com/madanalogy)**<br>
+Reviewers: [Pending](https://github.com/teammates)
+
+<box id="article-toc">
+
+* [Introduction to Security Testing](#introduction-to-security-testing)
+    * [What Is Security Testing?‎](#what-is-security-testing)
+    * [Why Have Security Testing?](#why-have-security-testing)
+    * [How to Get Started with Security Testing?](#how-to-get-started-with-security-testing)
+        * [Phase 1: Before Development Begins](#phase-1-before-development-begins)
+        * [Phase 2: During Definition and Design](#phase-2-during-definition-and-design)
+        * [Phase 3: During Development](#phase-3-during-development)‎
+        * [Phase 4: During Deployment‎](#phase-4-during-deployment)
+        * [Phase 5: Maintenance and Operations](#phase-5-maintenance-and-operations)
+    * [Additional Resources‎](#additional-resources)
+</box>
+
+## What Is Security Testing?
+
+Security Testing is **a type of Software Testing** that uncovers vulnerabilities, threats, risks in a software application and prevents malicious attacks from intruders. The purpose of Security Tests is to identify all possible loopholes and weaknesses of the software system which might result in a loss of information, revenue, repute at the hands of the employees or outsiders of the Organization.
+
+The goal of security testing is to:
+- Identify the threats in the system.
+- To measure the potential vulnerabilities of the system.
+- To help in detecting every possible security risks in the system.
+- To help developers in fixing the security problems through coding.
+
+:fas-info-circle: Knowledge of Software Development Life-Cycles (SDLCs) would be useful but not necessary.
```

Comment:
can utilise mark bind's `box type="info"` instead.

also can give a link to SDLC, either to internal chapters (if any) or external link

Reply:
> can utilise mark bind's box type="info" instead.

Thanks for the info, didn't know about the box type haha

> also can give a link to SDLC, either to internal chapters (if any) or external link

I was hoping this wouldn't be necessary,  but yes maybe I could put a link here to the "Additional Resources" section and have one there?

Reply:
> I was hoping this wouldn't be necessary, but yes maybe I could put a link here to the "Additional Resources" section and have one there?

Perhaps can put a hyperlink to the Software Development Life-Cycles text? in this way, readers can straight away navigate to explore more about that, instead of scrolling all the way down to additional resources to find out.

Reply:
As mentioned in the other comment, will proceed with a tooltip :)

Reply:
Okay, after trying out the different implementations, a link makes the most sense. I'll use a link to the CS2103 textbook.